### PR TITLE
Update governance rules with extended version

### LINKED ISSUE #6: Approve the Project Charter drafted at the Kickoff Workshop
# Purpose

The purpose of the project is to enable useful interoperability between Flow-based platforms, and to incentivize the joining of future software tools into an interoperable ecosystem.  We will accomplish this through a set of open specifications, and a set of open-source toolsets that reduce barriers to adoption.  The initial focus is on tools for mobile/web engagement.
# Members

The initial collaborating organisations are Nyaruka, InSTEDD, Ona, and VOTO Mobile.  UNICEF and USAID is providing guidance and input.  The project received initial funding from USAID. We recognize and hope that this project will outlast USAID funding and grow beyond the initial set of collaborating organisations.

# Scope
- We will create an open specification that allows sharing flow definitions across platforms, so that flows can be authored, run, and visualized across platforms
- We will create an open specification for the data resulting from interacting with flows, so that data can move across platforms and accelerate the creation of dashboards, analysis tools, and archiving.
- We will create a set of supporting tools to incentivize and accelerate adoption of these two specifications.

# Principles

The principles of the project will guide and be used as a criteria for decision-making:
- Think about and enable a future ecosystem. 
- Do the hard work to make it simple
- Start with needs 
- Focus on what’s good for end-users, within our context
- There are multiple good solutions; 
- Aim for consistency when possible but recognize unique needs
- Multiple perspectives are an asset
- “This will do”
- Make evidence-based decisions
- Understand that iteration will be needed (short-term, but also in the long-term to stay relevant and adapt)
- Make it open [includes how the initial partners work together, but also how others can join]

# Governance and Decision-making on the overall project:

(Includes specifications, recommendations for APIs, design of tools/toolsets)
- There is a ‘core’ team (for this: Nyaruka, InSTEDD, Ona, VOTO on all components; additionally UNICEF is on the Flow Data/Results Spec)
  - Criteria for being part of the core team:
    - Doers, not just users
    - Are (or soon will be) an implementor of the spec in some way
- We will operate on a consensus model: Discuss until all agree on the change, or one core member walks away. The goal is to avoid anyone walking away: we share a commitment to ensuring “this will work” for everyone
  - Consensus doesn’t mean agreement that a proposal is ideal; it means willingness to move forward with it
  - If we notice that one member is continuously dissenting but giving a final go-ahead: this is a red flag that either:
    - One partner is less aligned on the direction of the project
    - Or one voice is consistently not being heard
- Pull requests for incorporation into the spec require “all to approve” among the core team
  - The implicit default if no response is received in a week, or no official position given in a month, is to abstain.
- Anyone in the world can contribute to a branch
- Core team is the ones that can merge into master
- Where/how the discussion happens:
  - Asynchronous conversations on Github Issues and pull requests
  - Bi-weekly phone calls with the core team (and open to anyone), to discuss the tickets that need discussion
- In each ‘approval’ process, there is one vote per core organization. The vote choices are: Approve, Abstain, Disagree but Allow, Block
  - Agree: The org thinks the change is positive and approves it.
  - Abstain: The org doesn't have a position on this change, but approves it.
  - Disagree but Allow: The org thinks the change is negative but is willing to approve it.
  - Block: The org thinks the change is negative and is not willing to approve it until concerns are addressed.

### PR DESCRIPTION
**Pull request by markboots**
This updates the governance rules using the extended version discussed in the DC workshop. Fixes #6.

### CODE REVIEW

File: charter.md [hunk 1]
```diff
@@ -1,12 +1,57 @@
...
- 
+# Governance Rules
+
+## Purpose
+
+The purpose of the project is to enable useful interoperability between Flow-based platforms, and to incentivize the joining of future software tools into an interoperable ecosystem.  We will accomplish this through a set of open specifications, and a set of open-source toolsets that reduce barriers to adoption.  The initial focus is on tools for mobile/web engagement.
+
+## Members
+
+The initial collaborating organisations are Nyaruka, InSTEDD, Ona, and VOTO Mobile.  UNICEF and USAID is providing guidance and input.  The project received initial funding from USAID. We recognize and hope that this project will outlast USAID funding and grow beyond the initial set of collaborating organisations.
```
Comment (pld) [inline charter.md]: is => are providing...

File: charter.md [hunk 2]
```diff
@@ -1,12 +1,57 @@
...
+
+## Scope
+
+- We will create an open specification that allows sharing flow definitions across platforms, so that flows can be authored, run, and visualized across platforms
```
Comment (pld) [inline charter.md]: add period for consistency

File: charter.md [hunk 3]
```diff
@@ -1,12 +1,57 @@
...
+- We will create an open specification for the data resulting from interacting with flows, so that data can move across platforms and accelerate the creation of dashboards, analysis tools, and archiving.
+- We will create a set of supporting tools to incentivize and accelerate adoption of these two specifications.
+
+## Principles
+
+The principles of the project will guide and be used as a criteria for decision-making:
+
+- Think about and enable a future ecosystem.
```
Comment (pld) [inline charter.md]: perhaps these should be numbered so we can reference them by number elsewhere when justify, defending, and promoting specific decisions

**Comment by markboots** [comment]
Hi @pld , changes reflected ; let me know what you think! @edjez we need approval from your team as well. Thanks!

**Comment by markboots** [comment]
Hi @pld , would be grateful if you could OK your requested changes so we can merge this... Thanks!

**Pull request merged**